paasta_tools/mesos_tools.py

```diff
@@ -111,12 +111,16 @@ def is_mesos_leader(hostname=MY_HOSTNAME):
     return get_mesos_leader() == hostname
 
 
-def get_current_tasks(job_id):
+def get_current_tasks(job_id, include_orphans=False):
     """ Returns a list of all the tasks with a given job id.
     :param job_id: the job id of the tasks.
     :return tasks: a list of mesos.cli.Task.
     """
-    return get_mesos_master().tasks(fltr=job_id, active_only=False)
+    mesos_master = get_mesos_master()
+    framework_tasks = mesos_master.tasks(fltr=job_id, active_only=False)
+    if include_orphans:
+        framework_tasks += mesos_master.orphan_tasks()
```

Comment:
but now the filtering doesn't do the correct thing? This will return all running tasks matching the filter, plus *all* the orphans if you set ``include_orphans=True``?